packages/flutter_test/test_fixes/flutter_test/semantics_controller.dart.expect

```diff
@@ -0,0 +1,19 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // Generic reference variables.
+  finders.FinderBase<Element> theStart;
+  finders.FinderBase<Element> theEnd;
+
+  testWidgets('simulatedAccessibilityTraversal', (WidgetTester tester) async {
+    // Changes made in https://github.com/flutter/flutter/pull/143386
+    tester.semantics.simulatedAccessibilityTraversal();
+    tester.semantics.simulatedAccessibilityTraversal(startNode: theStart);
```

Comment:
Hm, this will not compile, though, because `startNode` expects a `FinderBase<SemanticsNode>`, but `theStart` is a `FinderBase<Element>`.

Is this intentionally a part-fix? I am not sure how helpful that is...

Reply:
It is partial. I always prefer to provide when we can, because during the deprecation period, the analyzer will say "Use startNode instead."
But after we remove it, it will only say "This is undefined."

Having the dart fix here means they will at least have a hint in the latter scenario that says "Use startNode."